src/Page/Food/Builder.elm

```diff
@@ -91,6 +94,7 @@ type Msg
     | DeletePreparation Preparation.Id
     | LoadQuery Query
     | NoOp
+    | OnHover (List (CI.Many ComparativeChart.Entry CI.Any))
```

Comment:
Maybe name this msg slightly more explicitely eg. `OnChartHover` or `OnComparativeChartHover`?